terraform/azure/projects/carbonplan.tfvars

```diff
@@ -0,0 +1,63 @@
+subscription_id    = "c5e7a734-3dbf-4285-80e5-4c0afb1f65dc"
+resourcegroup_name = "2i2c-carbonplan-cluster"
+
+ssh_pub_key = "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQCrtD6l/2S2dDT1OFLIUa46KAuzlUkckneQ4oyK5c2izT0nlBOxOcWACvnJ4rqntZVuhML7GYS35tQQnXyEXctTS2LnB7illl+oCT63WQzhhutfPLZSW3gJ+/CFVVp0VdHq4t1O8CQO+EgeV1cvuBre/neNZo5tcqXEsoV7tw/qw8XmD7Dk3yPz4NEFYZyZ4xQM7Qn2j9krZlRO2wCrosMFLo5Rv108PSDTHn3VtXEpJwemUO93D0ipJvCDE62M3vJxamqP4k5HLVn3Jun5KEvA7fINvgP3NSXkPVEEE4Prcqc1IJIzSxzygEN7xkZYsUEfJkybGXUWtUXdNjOEfLLlhYToMhdcP4jZluAjTtQmnJTr1sLegH+kbGIvI8MFIJAgXXLXS8/ubXHMozb+4gXRhyT8SMPwmgXGA+FtDduy5gnpPMSgmozQY17av5r25T6viWX5ivtL+n/CNJGf4npM1vKWcUNKiGyJrwzxQdpwJZ2uMP3EIdCYxOvqCQ+7UEM= sgibson@Athena.broadband"
+
+global_container_registry_name = "2i2ccarbonplanhubregistry"
+global_storage_account_name = "2i2ccarbonplanhubstorage"
+
+notebook_nodes = {
+  "small" : {
+    min : 0,
+    max : 20,
+    vm_size : "Standard_E2s_v4"
+  },
+  "medium" : {
+    min : 0,
+    max : 20,
+    vm_size : "Standard_E4s_v4"
+  },
+  "large" : {
+    min : 0,
+    max : 20,
+    vm_size : "Standard_E8s_v4"
+  },
+  "huge" : {
+    min : 0,
+    max : 20,
+    vm_size : "Standard_E32s_v4"
+  },
+  "vhuge" : {
+    min : 0,
+    max : 20,
+    vm_size : "Standard_M64s_v2"
+  },
+  "vvhuge" : {
+    min : 0,
+    max : 20,
+    vm_size : "Standard_M128s_v2"
+  },
+}
+
+dask_nodes = {
```

Comment:
In terraform, let's just leave this undefined - I think that should make it default to same set of sizes as notebook nodes.

Reply:
I took my lead from the jsonnet file where there seem to be fewer dask nodes than notebook nodes? https://github.com/2i2c-org/infrastructure/blob/aaf5534a9fca24bc56cef9cae9ea3083d4f8cc2f/eksctl/carbonplan.jsonnet#L33-L38

Reply:
Hmm, good point. I think keeping the notebook nodes match the dask nodes will make things easier to maintain for us. I'll work on fixing the jsonnet on eksctl to match that as well!

While looking through the code, I discovered that in our Azure terraform code, we don't actually default to dask nodes being same as notebook nodes! So I made https://github.com/2i2c-org/infrastructure/pull/834

Reply:
I removed the dask nodepools and updated against `master`. Will go for a deploy!